parcels/include/parcels.h

```diff
@@ -73,6 +73,65 @@ static inline ErrorCode spatial_interpolation_bilinear(double xsi, double eta, f
   return SUCCESS;
 }
 
+/* Bilinear interpolation routine for 2D grid for tracers with inverse distance weighting near land*/
+static inline ErrorCode spatial_interpolation_bilinear_invdist_land(double xsi, double eta, float data[2][2], float *value)
```

Comment:
Looks very good to me - also the inverse distance-weighting scheme for the interpolation is a nice idea in this sort of scenario.